doc/integrator/operations.rst

```diff
@@ -3,8 +3,10 @@
 Application Operation
 ---------------------
 
-This chapter describes features and configurations that are useful
-for IT-operations.
+This chapter describes features and configurations that are useful for IT-operations.
+
+On ``<application base URL>/c2c`` you have a page with all the services usefull for an administrator.
```

Comment:
At ``<application base URL>/c2c``, you have a page containing services useful for an administrator.